File: mystic-tales-podcast-frontend/mystic-tales-podcast-console/src/core/types/podcast-show.ts
export type PodcastCategory = {
  Id: number;
  Name: string;
};

export type PodcastSubCategory = {
  Id: number;
  Name: string;
  PodcastCategoryId: number;
};

export type PodcastShowsSubscriptionType = {
  Id: number;
  Name: string;
};

export type PodcastSubscriptionCycleType = {
  Id: number;
  Name: string;
};

export type PodcastSubscriptionBenefit = {
  Id: number;
  Name: string;
};

export type PodcastSubscriptionCycleTypePrice = {
  PodcastSubscriptionId: number;
  SubscriptionCycleType: PodcastSubscriptionCycleType;
  Version: number;
  Price: number;
  CreatedAt: string;
  UpdatedAt: string;
};

export type PodcastSubscriptionBenefitMapping = {
  PodcastSubscriptionId: number;
  PodcastSubscriptionBenefit: PodcastSubscriptionBenefit;
  Version: number;
  CreatedAt: string;
  UpdatedAt: string;
};

export type ShowSubscription = {
  Id: number;
  Name: string;
  Description: string;
  PodcastChannelId: string;
  PodcastShowId: string;
  IsActive: boolean;
  CurrentVersion: number;
  DeletedAt: string;
  CreatedAt: string;
  UpdatedAt: string;
  PodcastSubscriptionCycleTypePriceList: PodcastSubscriptionCycleTypePrice[];
  PodcastSubscriptionBenefitMappingList: PodcastSubscriptionBenefitMapping[];
};

export type PodcastEpisodeSubscriptionType = {
  Id: number;
  Name: string;
};

export type ShowEpisode = {
  Id: string;
  Title: string;
  Description: string;
  ExplicitContent: boolean;
  ReleaseDate: string;
  IsReleased: boolean;
  MainImageFileKey: string;
  AudioFileKey: string;
  AudioFileSize: number;
  AudioLength: number;
  AudioFingerprint: string;
  PodcastEpisodeSubscriptionType: PodcastEpisodeSubscriptionType;
  PodcastShowId: string;
  SeasonNumber: number;
  TotalSave: number;
  ListenCount: number;
  IsAudioPublishable: boolean;
  TakenDownReason: string;
  DeletedAt: string;
  CreatedAt: string;
  UpdatedAt: string;
};

export type PodcastShow = {
  Id: number;
  Name: string;
  Description: string;
  Language: string;
  ReleaseDate: string;
  Copyright: string;
  UploadFrequency: string;
  AverageRating: number;
  RatingCount: number;
  MainImageFileKey: string;
  TrailerAudioFileKey: string;
  TotalFollow: number;
  ListenCount: number;
  PodcasterId: number;
  PodcastCategory: PodcastCategory;
  PodcastSubCategory: PodcastSubCategory;
  PodcastShowsSubscriptionType: PodcastShowsSubscriptionType;
  PodcastChannnelId: number;
  TakenDownReason: string;
  DeletedAt: string;
  CreateAt: string;
  UpdatedAt: string;
  ShowSubscriptionList: ShowSubscription[];
  ShowEpisodeList: ShowEpisode[];
};

File: mystic-tales-podcast-backend/podcast-service/PodcastService.DataAccess/Entities/SqlServer/PodcastShow.cs
using System;
using System.Collections.Generic;

namespace PodcastService.DataAccess.Entities.SqlServer;

public partial class PodcastShow
{
    public Guid Id { get; set; }

    public string Name { get; set; } = null!;

    public string Description { get; set; } = null!;

    public string Language { get; set; } = null!;

    public DateOnly? ReleaseDate { get; set; }

    public bool? IsReleased { get; set; }

    public string Copyright { get; set; } = null!;

    public string? UploadFrequency { get; set; }

    public double AverageRating { get; set; }

    public int RatingCount { get; set; }

    public string? MainImageFileKey { get; set; }

    public string? TrailerAudioFileKey { get; set; }

    public int TotalFollow { get; set; }

    public int ListenCount { get; set; }

    public int PodcasterId { get; set; }

    public int? PodcastCategoryId { get; set; }

    public int? PodcastSubCategoryId { get; set; }

    public int PodcastShowSubscriptionTypeId { get; set; }

    public Guid? PodcastChannelId { get; set; }

    public string? TakenDownReason { get; set; }

    public DateTime? DeletedAt { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }

    public virtual PodcastCategory? PodcastCategory { get; set; }

    public virtual PodcastChannel? PodcastChannel { get; set; }

    public virtual ICollection<PodcastEpisode> PodcastEpisodes { get; set; } = new List<PodcastEpisode>();

    public virtual ICollection<PodcastShowHashtag> PodcastShowHashtags { get; set; } = new List<PodcastShowHashtag>();

    public virtual ICollection<PodcastShowReview> PodcastShowReviews { get; set; } = new List<PodcastShowReview>();

    public virtual ICollection<PodcastShowStatusTracking> PodcastShowStatusTrackings { get; set; } = new List<PodcastShowStatusTracking>();

    public virtual PodcastShowSubscriptionType PodcastShowSubscriptionType { get; set; } = null!;

    public virtual PodcastSubCategory? PodcastSubCategory { get; set; }
}
